src/amp-story-player-impl.js

```diff
@@ -449,6 +450,18 @@ export class AmpStoryPlayer {
     return parseUrlWithA(this.cachedA_, inputUrl);
   }
 
+  /**
+   * @param {string} href
+   */
+  getCacheUrl_(href) {
```

Comment:
+1 for keeping PRs small but I think we should still ship things that work. This PR loads and runs more code, but doesn't add any feature.
Should we have one PR to add the dependency (this one), but keep this code for a separate one that adds the feature?

I don't feel strongly so I'm still approving it, up to you :)